addon/data/shoot-panel.js

```diff
@@ -173,7 +173,7 @@ class ShootPanel extends React.Component {
   }
 
   onLinkClick(e) {
-    self.port.emit("openLink", this.props.shot.viewUrl, "normal");
+    self.port.emit("openLink", this.props.shot.viewUrl, this.props.loadReason, "normal");
```

Comment:
I think the problem is that there are two openLink functions with two different signatures, one in the panel and one in the recall panel.